src/bci_build/package/valkey.py

```diff
@@ -49,6 +49,7 @@
         ),
         custom_end=textwrap.dedent(
             f"""
+            {DOCKERFILE_RUN} sed -e 's/^protected-mode yes/protected-mode no/' -e 's/^bind .*//' < /etc/valkey/default.conf.example > /etc/valkey/valkey.conf
```

Comment:
```suggestion
            {DOCKERFILE_RUN} while IFS= read -r line; do \
              line="${line/#protected-mode yes/protected-mode no}"; \
              line="${line/#bind /}"; \
              printf '%s\n' "$line"; \
            done < /etc/valkey/default.conf.example > /etc/valkey/valkey.conf
```

Reply:
theoretically, RUN statements should be posix shell and this isn't posix. given that sed is not that big, do we need to optimize it that way? I don't expect this to be a perfect permanent fix anyway.

Reply:
What's not POSIX about this? It does the job with Tumbleweed's sh

Reply:
> What's not POSIX about this? 

```
line="${line/#protected-mode yes/protected-mode no}";
```

there is no shell builtin string replacement in posix sh. and Tumbleweeds "sh" is bash in bash mode, not in posix mode.